packages/@okta/vuepress-site/docs/guides/okta-account-management-policy/main/index.md

```diff
@@ -0,0 +1,403 @@
+
+Before you add this rule, change the access control settings in your [password policy](/docs/concepts/policies/#password-policies).
+
+Send a PUT request to the `/api/v1/policies/{policyId}/rules/{ruleId}` endpoint. Include the following parameters in the request body:
+
+- `"accessControl": "AUTH_POLICY"`
+- `"type": "PASSWORD"`
+
+Repeat for any other password policy rules.
+
+> **Note:** For each rule, verify that `selfServicePasswordReset` and `selfServiceUnlock` are set to `ALLOW`. If they aren't, add a rule that specifically allows them. New rules that you add to your org default to `"accessControl": "AUTH_POLICY"`.
+
+```bash
+curl --location --request PUT '{yourSubdomain}/api/v1/policies/{policyId}/rules/{ruleId}' \
+--header 'Accept: application/json' \
+--header 'Content-Type: application/json' \
+--header 'Authorization: SSWS {apiToken}' \
+--data '{
+    "id": "{ruleId}",
+    "name": "sdfsd",
+    "priority": 1,
+    "status": "ACTIVE",
+    "conditions": {
+        "people": {
+            "users": {
+                "exclude": []
+            }
+        },
+        "network": {
+            "connection": "ANYWHERE"
+        }
+    },
+    "actions": {
+        "passwordChange": {
+            "access": "ALLOW"
+        },
+        "selfServicePasswordReset": {
+            "access": "ALLOW",
+            "requirement": {
+                "primary": {
+                    "methods": [
+                        "email"
+                    ]
+                },
+                "stepUp": {
+                    "required": false
+                },
+                "accessControl": "AUTH_POLICY"
+            }
+        },
+        "selfServiceUnlock": {
+            "access": "ALLOW"
+        }
+    },
+    "system": false,
+    "type": "PASSWORD"
+}'
+```
+
+### Example password policy update request
+
+Continue to use the same value for `policyId`.
+
+Set the value of `priority` above the catch-all rule but below the first [phishing-resistant authenticator](#add-a-rule-for-your-first-phishing-resistant-authenticator) (if you added it). Make sure that the first phishing-resistant authenticator rule stays at priority 1.
+
+See [Add a rule for authenticator enrollment](#add-a-rule-for-authenticator-enrollment).
+
+### User experience
+
+There are no changes to the user experience when you move password recovery and account unlock to the account management policy. However, be aware of how the account management policy works with the following features:
+
+- [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in): Works with the account management policy if you configure the authentication frequency correctly. The **Prompt for authentication** setting must be more frequent than the equivalent setting in your Okta End-User Dashboard authentication policy. Setting **Prompt for authentication** in your Okta account management policy to every time ensures that users don't have to wait to reset a password.
```

Comment:
```suggestion
- [Stay signed in](https://help.okta.com/okta_help.htm?type=oie&id=ext-stay-signed-in): Works with the account management policy if you configure the authentication frequency correctly. The **Prompt for authentication** setting must be more frequent than the equivalent setting in your Okta Dashboard authentication policy. Setting **Prompt for authentication** in your Okta account management policy to every time ensures that users don't have to wait to reset a password.
```

Reply:
I double checked this in Trex and it should actually just be Okta Dashboard (not End-User). The app name is Okta Dashboard.
![image](https://github.com/user-attachments/assets/f44a05cf-58a0-4921-8c7a-bdcb4c1a1e41)